tools/sgcollect_info

```diff
@@ -777,9 +773,9 @@ def main():
             do_upload_and_exit(zip_filename, upload_url)
 
     if options.redact_level != "none":
-        print "Zipfile built: {0}".format(redact_zip_file)
+        print("Zipfile built: {0}".format(redact_zip_file))
 
-    print "Zipfile built: {0}".format(zip_filename)
+    print("Zipfile built: {0}".format(zip_filename))
```

Comment:
Ditto on python version syntax check.

Reply:
It's Python 3 syntax but also backwards compatible with 2.x, so long as there's only a single argument.